src/main/java/tat/framework/screen/BasePage.java

```diff
@@ -0,0 +1,26 @@
+package tat.framework.screen;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class BasePage {
+
+    protected static WebDriver driver;
+
+    public BasePage(){
+        driver = setUpBrowser();
+        PageFactory.initElements(driver, this);
+    }
+
+    public static WebDriver setUpBrowser(){
+        if (driver == null){
+            WebDriverManager.chromedriver().setup();
+            driver = new ChromeDriver();
+            driver.manage().window().maximize();
+        }
+        return driver;
+    }
```

Comment:
Get rid of this method. Just use getDriver() method from DriverSingleton. See comment about Singleton pattern